thrift/thrift0.13/src/main/java/com/linecorp/armeria/internal/server/thrift/ThriftDocServicePlugin.java

```diff
@@ -298,20 +303,31 @@ private static DescriptiveTypeInfo newDescriptiveTypeInfo(
     public static final class Entry {
```

Comment:
This isn't your fault, but let's remove `public` modifier here:
```suggestion
    static final class Entry {
```